indico/modules/events/registration/util.py

```diff
@@ -493,3 +496,31 @@ def generate_ticket(registration):
 
 def get_ticket_attachments(registration):
     return [('Ticket.pdf', generate_ticket(registration).getvalue())]
+
+
+def import_registrations_from_csv(regform, fileobj, skip_moderation=True, notify_users=False):
+    """Import event registrants from a CSV file into a form."""
+    reader = csv.reader(fileobj)
+    registrations = []
+    for n_row, row in enumerate(reader, 1):
+        try:
+            first_name, last_name, affiliation, position, phone, email = [to_unicode(value).strip() for value in row]
+        except ValueError:
+            raise UserValueError(_('Row {}: malformed CSV data - please check that the number of columns is correct')
+                                 .format(n_row))
+
+        if not email:
+            raise UserValueError(_('Row {}: missing e-mail address').format(n_row))
+        if not first_name or not last_name:
+            raise UserValueError(_('Row {}: missing first or last name').format(n_row))
+
+        with db.session.no_autoflush:
+            registrations.append(create_registration(regform, {
+                'email': email,
+                'first_name': first_name.title(),
+                'last_name': last_name.title(),
+                'affiliation': affiliation,
+                'phone': phone,
+                'position': position
+            }, notify_user=notify_users, management=skip_moderation))
```

Comment:
Using the `management` kwarg also affects e.g. the log realm. Maybe better to add a new `skip_moderation` kwarg?